### ISSUE TITLE
Alternative way to display language tabs

### ISSUE THREAD

**Issue by ninjasitm** [optimistdigital/nova-translatable#55]
Hello,

Firstly, thank you for this amazing nova package. I've been using it on a current project and it is a lifesaver.

I would like to propose an option for displaying language tabs. Currently, if I have too many languages enabled I get a layout similar to the following:
![image](https://user-images.githubusercontent.com/4931388/119013051-9b66b600-b964-11eb-8f06-3402402332f6.png)

If I use the CSS below:
```
. translatable-field > .flex > .ml-auto {
    margin-left: auto;
    display: flex;
    flex-direction: column;
}
```
I get:
![image](https://user-images.githubusercontent.com/4931388/119012958-825e0500-b964-11eb-9782-1e803c7c99eb.png)

# Question
Is it possible to give an option to change the layout of the fields. For example as either a dropdown, a row, or a column? I can provide a pull request if you think that's necessary?

**Comment by Tarpsvo** [comment]
Hi! I added a new `display_type` configuration option for this in version 1.9.0.

It currently supports two options: 'row' and 'column'.

If you want to set it to column, you have two options:
a) Set it in the `config/nova-translatable.php` to 'column', as shown here:
https://github.com/optimistdigital/nova-translatable/blob/6d81745270a20db85ab3f10b098633511349d23f/config/nova-translatable.php#L31

b) Set it on a per-field basis using the second parameter of the ->translatable() function:
```
Text::make('Some field')
  ->translatable(null, ['displayType' => 'column']),
```

I don't have time to make it a dropdown right now, but you are welcome to make a PR for that. :) I already added all the configurability for it, so all the values required (locales, displayType) should be available in LocaleTabs.vue.

Hope this helps! Good luck!
